plugins/modules/hcloud_load_balancer_target.py

```diff
@@ -226,13 +226,13 @@ def _create_load_balancer_target(self):
         if not self.module.check_mode:
             try:
                 self.hcloud_load_balancer.add_target(**params).wait_until_finished()
-            except APIException as e:
-                if e.code == "locked" or e.code == "conflict":
+            except APIException as exception:
```

Comment:
Pro tip: embed actual context into the variable names. I tend to do this, for example:
```suggestion
            except APIException as api_err:
```